tests/test_halo_update.py

```diff
@@ -310,7 +311,7 @@ def test_halo_update_timer(
         req_list.append(req)
     for req in req_list:
         req.wait()
-    required_times_keys = ("pack", "unpack", "Isend", "Recv")
+    required_times_keys = ("pack", "unpack", "Isend", "Irecv", "wait")
```

Comment:
The timers are now called `Irecv` instead of `Recv` since the halo update does a `Irecv`. And I time the `wait` of the request separately as well. The utest checks the key that are recoded

Reply:
OK cool. Can you add this change to the PR description?

Reply:
Updated, adding also a couple of other changes.